src/ctia/schemas/search_agg.clj

```diff
@@ -30,7 +30,14 @@
   (st/optional-keys
    {:filter-map   {s/Keyword s/Any}
     :range        RangeQuery
-    :full-text    FullTextQuery}))
+    :full-text    [FullTextQuery]}))
+
+(s/defschema ESQFullTextQuery
```

Comment:
While `SearchQuery` is a generic model which could be used for a potential other store implementation, this new schema is really specific to and does more belong to `ctia.stores.es`.
Indeed, it should even go into `ductile`, but do not for that far, I have some other schemas which should go into ductile: https://github.com/threatgrid/ctia/blob/944cbd0b05a64019dc3154dea3fb2aef8266d5a7/src/ctia/stores/es/crud.clj#L223
Then, you can either add this schema in `ctia.stores.es.crud` or create a `ctia.stores.es.schemas` ns in which you could also move `ESActionResult` and `ESBulkRes`.